public/javascripts/Progress/src/Progress.js

```diff
@@ -57,4 +57,39 @@ function Progress (_, $, difficultRegionIds, userRole) {
         InitializeSubmittedLabels(map, streetParams, 'null', InitializeMapLayerContainer(), data2[0])
         setRegionFocus(map, layers);
     });
+
+    function logWebpageActivity(activity){
+        var url = "/userapi/logWebpageActivity";
+        var async = false;
+        $.ajax({
+            async: async,
+            contentType: 'application/json; charset=utf-8',
+            url: url,
+            type: 'post',
+            data: JSON.stringify(activity),
+            dataType: 'json',
+            success: function(result){},
+            error: function (result) {
+                console.error(result);
+            }
+        });
+    }
+
+    function putUserOrg(e) {
+        var orgId = $(this).attr('id');
+        $.ajax({
+            async: true,
+            url: '/userapi/setUserOrg/' + orgId,
+            type: 'put',
+            success: function (result) {
+                window.location.reload();
+                logWebpageActivity("Click_module=EditedOrg");
```

Comment:
Can you include the org they are joining? If it's easy, including the org they are leaving would be nice as well!